tests/foreman/cli/test_discoveryrule.py

```diff
@@ -132,6 +128,9 @@ def test_positive_create_with_search(self, discoveryrule_factory):
         search_query = 'cpu_count = 2'
         rule = discoveryrule_factory(options={'search': search_query})
         assert rule.search == search_query
+        custom_query = 'processor = x86'
+        rule = discoveryrule_factory(options={'search': custom_query})
+        assert rule.search == custom_query
```

Comment:
Was this coverage added as an action item from the component evaluation or moved from another test? We are essentially repeating the test again just with a different query so it's a little repetitive.

Reply:
First search is for default queries or predefined queries and second is the custom query. It would be good to test both types.

Reply:
What is a default / predefined query?

Reply:
Default query -  Sugestions you get while creating a discovery rule on satellite webui.
custom query -  Any query that you wish to add that is not part of the predefined suggestions.
Yes, @Griffin-Sullivan it is a bit redundant as it was a suggestion for sd/dev to include some custom queries in our testing.

Reply:
Ok we can leave it, but really you would be more concerned about it in the UI tests then.